src/components/GraphFilter/GraphFilter.tsx

```diff
@@ -4,35 +4,22 @@ import { Toolbar, Button, ButtonGroup, Switch, Icon, FormGroup } from 'patternfl
 import { GraphFilterProps, GraphFilterState } from '../../types/GraphFilter';
 import { ToolbarDropdown } from '../ToolbarDropdown/ToolbarDropdown';
 import AutoUpdateNamespaceList from '../../containers/AutoUpdateNamespaceList';
+import { config } from '../../config';
 
 export default class GraphFilter extends React.Component<GraphFilterProps, GraphFilterState> {
   // TODO:  We should keep these mappings with their corresponding filtering components.
   // GraphFilter should be minimal and used for assembling those filtering components.
-  static INTERVAL_DURATION = [
-    [60, '1 minute'],
-    [600, '10 minutes'],
-    [1800, '30 minutes'],
-    [3600, '1 hour'],
-    [14400, '4 hours'],
-    [28800, '8 hours'],
-    [86400, '1 day'],
-    [604800, '7 days'],
-    [2592000, '30 days']
-  ];
-  static GRAPH_LAYOUTS = [
-    ['breadthfirst', 'Breadthfirst'],
-    ['cola', 'Cola'],
-    ['cose', 'Cose'],
-    ['dagre', 'Dagre'],
-    ['klay', 'Klay']
-  ];
+  static INTERVAL_DURATION = config().toolbar.intervalDuration;
```

Comment:
To have centralized this in a single place is good.
Just minor question due perhaps I'm not familiar with best practices.
Why a config() function instead of a plain exported object ?
(Non blocker).

Reply:
I saw in another projects that they have the config file like this maybe we need to add some methods in the configuration in a future.